src/libsystemd/sd-device/device-monitor.c

```diff
@@ -774,7 +774,7 @@ int device_monitor_send_device(
          * If we send to a multicast group, we will get
          * ECONNREFUSED, which is expected.
          */
-        smsg.msg_name = destination ? &destination->snl : &default_destination;
+        smsg.msg_name = (struct sockaddr_nl*) &(destination ?: &default_destination)->nl;
```

Comment:
Aren't you taking a double address of `default_destination` here`?

Reply:
I do not think so? If so, compile should fail and CIs will detect that.

Reply:
I got confused about operator precedence, nvm